src/seedu/addressbook/data/tag/TaggingOperationRecord.java

```diff
@@ -0,0 +1,40 @@
+package seedu.addressbook.data.tag;
+
+import java.util.ArrayList;
+
+import seedu.addressbook.data.person.Person;
+
+enum TaggingOperationType {
+    ADD, 
+    REMOVE
+}
+
+public class TaggingOperationRecord {
+	private Person person;
+	private Tag tag;
+	private TaggingOperationType type;
+	
+	static public ArrayList<TaggingOperationRecord> defaultRecordList = new ArrayList<TaggingOperationRecord>();
```

Comment:
This is not correct based on the class diagram. Remember that AddressBook should be the one that takes care of keeping track of the Tagging objects.